megalinter/descriptors/lua.megalinter-descriptor.yml

```
@@ -2,6 +2,14 @@ descriptor_id: LUA
 descriptor_type: language
 file_extensions:
   - ".lua"
+install:
+  dockerfile:
+    - |
+      RUN wget --tries=5 https://www.lua.org/ftp/lua-5.3.5.tar.gz -O - -q | tar -xzf - \
+          && cd lua-5.3.5 \
+          && make linux \
+          && make install \
+          && cd .. && rm -r lua-5.3.5/ \
```

```suggestion
          && cd .. && rm -r lua-5.3.5/
```
After that, rebuild (sync) the Dockerfiles and should fix the invalid build